plugin-server/src/worker/ingestion/person-state.ts

```diff
@@ -214,11 +215,15 @@ export class PersonState {
         }
 
         const update: Partial<Person> = {}
-        const updatedProperties = this.updatedPersonProperties(personFound.properties || {})
 
-        if (!equal(personFound.properties, updatedProperties)) {
-            update.properties = updatedProperties
+        const updatedProperties = this.updatedPersonProperties(personFound)
+        // ignore timestamp and operation only updates as we don't really use them yet
+        if (!equal(personFound.properties, updatedProperties.properties)) {
+            update.properties = updatedProperties.properties
+            update.properties_last_operation = updatedProperties.properties_last_operation
+            update.properties_last_updated_at = updatedProperties.properties_last_updated_at
```

Comment:
Behavior/edge cases around this are not properly covered in tests - e.g. missing test is when we $set again with same value the timestamp would not change/query is not made.